aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,97 @@
+        for origin in dist.get("Origins", {}).get("Items", []):
+            if origin.get("DomainName") in target_origins:
+                if dist.get("Staging", False):
+                    staging_dist = dist
+                    print(f"Found staging distribution: {dist['Id']}")
+                else:
+                    production_dist = dist
+                    print(f"Found production distribution: {dist['Id']}")
+                break
+
+    if not staging_dist or not production_dist:
+        raise Exception("Could not find both staging and production distributions for vilnacrmtest.com")
```

Ruff (0.11.9)

Create your own exception

(TRY002)